### PR TITLE
fix(doris) , mysql group_concat function different doris , it is not support separator keyword

### PR DESCRIPTION
**Pull request by Toms1999**
#3589

### CODE REVIEW

**Comment by tobymao** [comment]
add a test

File: sqlglot/dialects/doris.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ class Generator(MySQL.Generator):
             exp.ArrayUniqueAgg: rename_func("COLLECT_SET"),
             exp.CurrentTimestamp: lambda self, _: self.func("NOW"),
             exp.DateTrunc: lambda self, e: self.func("DATE_TRUNC", e.this, unit_to_str(e)),
+            exp.GroupConcat: lambda self,
+                                    e: f"""GROUP_CONCAT({self.sql(e, "this")} , {self.sql(e, "separator") or "','"})""",
```
Comment (tobymao) [inline sqlglot/dialects/doris.py:53]: exp.func("GROUP_CONCAT", e.this, e.args.get("seprator") or ",")

Comment (Toms1999) [reply sqlglot/dialects/doris.py]: yes , but need to add ''

Comment (tobymao) [reply sqlglot/dialects/doris.py]: exp.Literal.string(",")

**Comment by tobymao** [comment]
are you going to add a test and make style?

**Comment by Toms1999** [comment]
@tobymao 
I tried to do so, but it never passed. I don't know why. I'm sorry

**Pull request closed without merging**